sdk/netapp/Microsoft.Azure.Management.NetApp/src/Customizations/NetAppResourceOperationsExtensions.cs

```diff
@@ -51,7 +51,7 @@ public static partial class NetAppResourceOperationsExtensions
         /// <param name='resourceGroup'>
         /// Resource group name.
         /// </param>
-        //[System.Obsolete("This operation is deprecated. Please use CheckFilePathAvailability(string location, string name, string subnetId) instead.", false)]
+        [System.Obsolete("This operation is deprecated. Please use CheckFilePathAvailability(string location, string name, string subnetId) instead.", false)]
```

Comment:
The breaking change policy means that we should not obsolete this method, it shoudl continue to be supported.

Reply:
understood, just thought this would make it more visible to the user to use the new api.  Will change.